ngraph/frontend/paddlepaddle/src/model.cpp

```diff
@@ -289,7 +289,14 @@ InputModelPDPD::InputModelPDPDImpl::InputModelPDPDImpl(const std::basic_string<T
 
     FRONT_END_GENERAL_CHECK(pb_stream && pb_stream.is_open(), "Model file doesn't exist");
     FRONT_END_GENERAL_CHECK(m_fw_ptr->ParseFromIstream(&pb_stream), "Model can't be parsed");
-
+    // According to Paddle, the saved model has the framework version
+    // For exmaple Paddle 2.1.0 is encoded as 2001000. 0 means latest framework.
+    // https://github.com/PaddlePaddle/Paddle/blob/develop/cmake/version.cmake
+    // https://github.com/PaddlePaddle/Paddle/blob/2100816c5190693cc7dee181e96af72e9f0fbd1d/paddle/fluid/framework/program_desc.cc#L52
+    int64_t version = m_fw_ptr->version().version();
+    FRONT_END_GENERAL_CHECK(
+        version >= 2001000 || version == 0,
```

Comment:
I guess we can create test for this as well. As far as I remember @mvafin did some 'binary' modifications to simulate 'unsupported operation'. With same approach we could simulate 'unsupported version'

Reply:
You are right, Let me figure out a way to simulate a lower version model file

Reply:
Tests enabled in 7a9e55f6a867a653d004e2dd8e86e0cb0bf27e78